examples/cdn-api-reference/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import fastifyStatic from '@fastify/static'
 import apiReferenceBundle from '@scalar/api-reference/browser/standalone.js?raw'
+import apiReferenceComponentBundle from '@scalar/api-reference/component.js?raw'
```

Comment:
should we call this web-component or something to indicate it’s not a Vue component?

Reply:
Yeah or custom element maybe?

Reply:
Would work too!